EIPS/static_call.md

```diff
@@ -0,0 +1,41 @@
+## Preamble
+<pre>
+  EIP: to be assigned
+  Title: New opcode STATIC_CALL
+  Author: Vitalik Buterin &lt;vitalik@ethereum.org&gt;, Christian Reitwiessner &lt;chris@ethereum.org&gt;
+  Type: Standard Track
+  Category(*only required for Standard Track): Core
+  Status: Draft
+  Created: 2017-02-13
+</pre>
+
+## Simple Summary
+
+To increase smart contract security, this proposal adds a new opcode that can be used to call another contract (or itself) while disallowing any modifications to the state during the call (and its subcalls, if present).
+
+## Abstract
+
+
+## Motivation
+
+
+## Specification
+
+Opcode: `0xfa`.
+
+`STATIC_CALL` functions equivalently to a `CALL`, except it takes 6 arguments not including value, and calls the child with a `STATIC` flag on. Any calls, static or otherwise, made by an execution instance with a `STATIC` flag on will also have a `STATIC` flag on. Any attempts to make state-changing operations inside an execution instance with a `STATIC` flag on will instead throw an exception. These operations include nonzero-value calls, creates, `LOG` calls, `SSTORE`, `SSTOREBYTES` and `SUICIDE`.
```

Comment:
This was taken from vitalik's draft that assumed another EIP which introduced SSTOREBYTES.